src/compiler/scala/tools/reflect/package.scala

```diff
@@ -59,13 +58,13 @@ package object reflect {
     override def hasWarnings = reporter.hasWarnings
 
     def display(info: Info): Unit = info.severity match {
-      case API_INFO => reporter.info(info.pos, info.msg, force = false)
+      case API_INFO => reporter.echo(info.pos, info.msg)
       case API_WARNING => reporter.warning(info.pos, info.msg)
       case API_ERROR => reporter.error(info.pos, info.msg)
     }
 
     def interactive(): Unit = reporter match {
-      case reporter: AbstractReporter => reporter.displayPrompt()
+      case reporter: PrintReporter => reporter.displayPrompt()
```

Comment:
It's lunch time, just confirming that `interactive` was intentionally removed.

Reply:
Yes, i missed this one. The method is unused.

Reply:
Also it caused
```
+Caused by: java.lang.NullPointerException
+	at scala.tools.nsc.interactive.tests.InteractiveTestSettings.prepareSettings(InteractiveTestSettings.scala:50)
+	at scala.tools.nsc.interactive.tests.InteractiveTestSettings.prepareSettings$(InteractiveTestSettings.scala:39)
+	at scala.tools.nsc.interactive.tests.InteractiveTest.prepareSettings(InteractiveTest.scala:50)
+	at scala.tools.nsc.interactive.tests.core.PresentationCompilerInstance.compiler(PresentationCompilerInstance.scala:28)
+	at scala.tools.nsc.interactive.tests.core.PresentationCompilerInstance.compiler$(PresentationCompilerInstance.scala:27)
+	at scala.tools.nsc.interactive.tests.InteractiveTest.compiler$lzycompute(InteractiveTest.scala:50)
+	at scala.tools.nsc.interactive.tests.InteractiveTest.compiler(InteractiveTest.scala:50)
+	at scala.tools.nsc.interactive.tests.core.PresentationCompilerInstance.$init$(PresentationCompilerInstance.scala:23)
+	at scala.tools.nsc.interactive.tests.InteractiveTest.<init>(InteractiveTest.scala:51)
+	at Test$.<init>(Test.scala:3)
+	at Test$.<clinit>(Test.scala:3)
```